homework/zadanie4_five_philosophers.cpp

```diff
@@ -72,37 +56,62 @@ class CPhilosoph{
         }
     }
 };
+ 
+class CTable{
 
+    int numbers_of_forks_;
+    std::vector<std::thread> thread_pool_;
+    std::vector<std::string> names_of_philosophers_;
```

Comment:
IMO niepotrzebne. Od razu w do filozofów dawaj ich nazwy. A tak to masz ten wektor, który jest używany tylko do zainicjalizowania ich imion i do niczego więcej nie służy.

Reply:
vector names_of_philosophers_ usuniety,